websites/D/Disney+/dist/metadata.json

```diff
@@ -1,22 +1,29 @@
-    "en": "Disney+ is the exclusive, unlimited access home for of all the best from Disney, Pixar, Marvel, Star Wars & National Geographic.",
-    "nl": "Disney+ biedt exclusieve, onbeperkte toegang tot het beste van Disney, Pixar, Marvel, Star Wars en National Geographic.",
-    "fr": "Disney+ réunit Disney, Pixar, Marvel, Star Wars, National Geographic et encore plus dans une destination.",
-    "de": "Das Zuhause von Disney, Pixar, Marvel, Star Wars und National Geographic. Die neuesten Blockbuster, exklusive Disney+ Originals oder unvergessliche Klassiker. Es gibt immer etwas Neues zu entdecken!",
-    "it": "Scopri il meglio di Disney, Pixar, Marvel, Star Wars e National Geographic. Film, serie, produzioni originali e molto altro.",
-    "es": "El servicio de streaming de Disney, Pixar, Marvel, Star Wars y National Geographic."
+    "en": "Disney+ is the exclusive home for your favorite movies and TV shows from Disney, Pixar, Marvel, Star Wars, National Geographic, and Star. Start streaming today.",
+    "nl": "Disney+ biedt exclusieve, onbeperkte toegang tot het beste van Disney, Pixar, Marvel, Star Wars, National Geographic en Star. Begin meteen met streamen.",
+    "fr": "Retrouvez en exclusivité vos films et séries Disney, Pixar, Marvel, Star Wars, National Geographic et Star préférés sur Disney+. Commencez à regarder Disney+ dès aujourd'hui.",
+    "de": "Disney+ ist der exklusive Streaming-Service für deine Lieblingsfilme und -serien von Disney, Pixar, Marvel, Star Wars, National Geographic und Star. Noch heute mit dem Streamen beginnen.",
+    "it": "Disney+ è la dimora esclusiva dei tuoi film e show TV preferiti di Disney, Pixar, Marvel, Star Wars, National Geographic e Star. Inizia lo streaming oggi.",
+    "es": "Disney+ es la plataforma exclusiva para ver tus películas, series y programas de televisión favoritos de Disney, Pixar, Marvel, Star Wars, National Geographic y Star. Ver en streaming hoy mismo.",
+    "pt": "O Disney+ é a morada exclusiva das suas séries e filmes preferidos da Disney, Pixar, Marvel, Star Wars, National Geographic e Star. Comece já a ver.",
+    "no": "Disney+ er det eksklusive hjemmet for favorittfilmene og seriene dine fra Disney, Pixar, Marvel, Star Wars, National Geographic og Star. Begynn å strømme i dag.",
+    "fi": "Disney+ tarjoaa suosikkielokuviasi ja tv-ohjelmia yksinoikeudella tekijöiltä Disney, Pixar, Marvel, Star Wars, National Geographic ja Star. Aloita katselu tänään.",
+    "sv_SE": "Disney+ är det enda hemmet för dina favoritfilmer och tv-program från Disney, Pixar, Marvel, Star Wars, National Geographic och Star. Börja streama idag.",
+    "da_DK": "Disney+ er det eksklusive hjem for dine yndlingsfilm og -tv-serier fra Disney, Pixar, Marvel, Star Wars, National Geographic og Star. Begynd at se i dag.",
+    "pt_BR": "O Disney+ é o serviço exclusivo para ver os seus filmes e programas preferidos da Disney, Pixar, Marvel, Star Wars e National Geographic. Comece a assistir hoje mesmo."
```

Comment:
you know all of these languages or you just used translator? Remove them if I'm right

Reply:
These come straight from the Disney+ site (minus the mention of "Star" that I added). I changed my profile's language for each of the languages to get the strings from the source code.